components/widgets/forest-change/glad-alerts-daily/index.js

```diff
@@ -0,0 +1,290 @@
+import { fetchGLADLatest, fetchHistoricalGladAlerts } from 'services/analysis-cached';
+
+import {
+  POLITICAL_BOUNDARIES_DATASET,
+  GLAD_DEFORESTATION_ALERTS_DATASET
+} from 'data/layers-datasets';
+import {
+  DISPUTED_POLITICAL_BOUNDARIES,
+  POLITICAL_BOUNDARIES,
+  GLAD_ALERTS
+} from 'data/layers';
+
+import getWidgetProps from './selectors';
+
+export default {
+  widget: 'gladAlertsDaily',
+  title: 'Glad Alerts Count in {location}',
+  large: true,
+  refetchKeys: ['dataset', 'forestType', 'landCategory', 'confidence'],
```

Comment:
we will only want `forestType` and `landCategory` here

Reply:
amended!